src/test/java/org/folio/entlinks/controller/LinksSuggestionsIT.java

```diff
@@ -46,7 +44,7 @@ class LinksSuggestionsIT extends IntegrationTestBase {
 
   @BeforeAll
   static void prepare() {
-    setUpTenant();
+    setUpConsortium(TENANT_ID, List.of(), false);
```

Comment:
Why do we need this change? I expect this test to work in non-consortium tenant.